apps/web/src/common/modules/navigations/lnb/type.ts

```diff
@@ -14,6 +14,7 @@ export const MENU_ITEM_TYPE = {
     DIVIDER: 'divider',
     FAVORITE_ONLY: 'favorite-only',
     DROPDOWN: 'dropdown',
+    EXTRA: 'extra',
```

Comment:
Suggestion: How about adding a type named 'slot'? This way, we could use this type for ambiguous cases going forward, and assign slot names as given `id`.

Example:

At `LNB.vue`

```html
                <div v-else-if="menuData.type === MENU_ITEM_TYPE.SLOT"
                     :key="`${idx}-${getUUID()}`"
                     class="slot-menu-wrapper"
                >
                    <slot :name=`slot-${menuData.id}` />
                </div>
```

At `CostExplorerLNB.vue`

```typescript
 const showMoreExtraMenuSet: LNBMenu = [{
   id: 'show-more',
   type: 'slot',
        }];
```

```html
            <template #slot-show-more>
                <p-collapsible-toggle :is-collapsed.sync="state.showMoreQuerySetStatus" />
            </template>
```

Reply:
I think it's good idea!!!!

Reply:
Thanks!! I agree with you